test/functionalTest/cases/716_forward_registered_attributes/forward_attribute.test

```diff
@@ -0,0 +1,303 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
What about the similar functionality in the queryContext case (i.e. existing CE Ent1-AttrA and a registration for Ent1-AttrB, doing a query for Ent1-AttrB will forward operation to CPr instead of returning 404 Not CE found)?